mods/ra/rules/defaults.yaml

```diff
@@ -159,7 +159,13 @@
 
 ^AutoTargetGround:
 	AutoTarget:
+		AttackAnythingCondition: stance-attackanything
 	AutoTargetPriority@DEFAULT:
+		RequiresCondition: !stance-attackanything
+		ValidTargets: Infantry, Vehicle, Water, Defense
```

Comment:
`Ant`s are also `Infantry`, so that would be unnecessary bloat.  In TD `Tank`s are also `Vehicle`s, but it looks like that isn't true in RA.  That's a regression from #13383 then, which I will fix here.